index.js

```diff
@@ -36,9 +36,9 @@ exports.callMarketPlaceFetchNft = async (config, args) => {
 /**
  * Returns a promise containing the connected wallet's balance.
  */
-exports.getWalletBalance = async () => {
+exports.getWalletBalance = async (config) => {
   const sb = await seabug;
-  return sb.getWalletBalance();
+  return sb.callGetWalletBalance(config);
```

Comment:
might this fix #36 ?

Reply:
No it doesn't fix that one, for that one a different function `connectWallet` was removed that we need to add back

Reply:
gotcha